faros_event.sh

```diff
@@ -866,7 +853,17 @@ function addRunToData() {
                 "pipeline": $run_pipeline,
                 "organization": $run_org,
                 "source": $run_source
-            }' <<< $request_body
+            }' <<< "$request_body"
+        )
+    fi
+     if ! [ -z "$run_name" ]; then
```

Comment:
I think passing `run_name` was missing - so I added it.

Reply:
how do I test for it?

Reply:
We need to find all the mutations where this needs to be added as a parameter and add it to their request body. Also, unfortunately the rest endpoint mutations need to be changed first, otherwise they'll fail on unexpected param (see https://github.com/hasura/graphql-engine/blob/master/rfcs/rest-endpoints.md#response-codes)

Maybe we can make these changes in a separate PR(s).

Reply:
> how do I test for it?

The tests just compare the expected and actual payload to hasura rest endpoints.